src/Makefile

```diff
@@ -24,7 +24,18 @@ MULT_KERN = default
 DIAG_DUMMY = 
 
 #Include system-dependent variables
-include system.make
+#Get the system name automatically.
+SYSTEM = $(shell hostname -d | cut -d. -f1)
+ifneq ($(SYSTEM),)
+SYSTEM := .$(SYSTEM)
+endif
+SYSTEM_PATH = system/system$(SYSTEM).make
+ifneq ("$(wildcard $(SYSTEM_PATH))","")
+$(info Building on SYSTEM $(SYSTEM), using makefile $(SYSTEM_PATH))
+include $(SYSTEM_PATH)
+else
+$(error Cannot find $(SYSTEM_PATH) file for system $(SYSTEM). Please make one, using system/system.example.make as an example)
```

Comment:
This is pretty minor, but since you've prefixed `SYSTEM` with a `.` it gets printed as `.<system name>` here